wire-ios/Wire-iOS/Sources/Helpers/SaveFileManager.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import UIKit
+import WireSystem
+
+protocol SaveFileActions {
+    func save(value: String, fileName: String, type: String)
```

Comment:
As far as I understand the function does not save the certificate itself, but presents the prompt to save for the user? So maybe `presentSaveDialog` or similar is more precise?

Reply:
This is a save action. The way it is handled is upto the manager. Right now we show a dialog to save it in Files which needs . This could change later to may be save with thin the app bundle. I guess having `save` as the action should be fine here.

Reply:
Okay